modular_skyrat/modules/customization/modules/surgery/organs/genitals.dm

```diff
@@ -184,8 +185,11 @@
 
 /obj/item/organ/genital/vagina/get_description_string(datum/sprite_accessory/genital/gas)
 	var/returned_string = "You see a [lowertext(genital_name)] vagina."
-	if(lowertext(genital_name) == "cloaca")
+	if(gas && gas.desc_string)
+		returned_string = gas.desc_string
 		returned_string = "You see a cloaca." //i deserve a pipebomb for this
```

Comment:
This line no longer necessary because we pull this from the sprite accessory object (gas)